index.html

```diff
@@ -237,4 +246,4 @@ <h2 class="Contact">Contact me</h2>
     </section>
```

Comment:
- [ ] Great job :rocket: implementing the previous reviewer's suggestions, but, unfortunately, the Contact Me section isn't matching the [Figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=0-1&t=MP8F21NcNgsWpIRH-0). The button hasn't been centered properly nor does the paragraph has the correct width.

Reply:
Addressed the issue